admin_guide/manage_rbac.adoc

```diff
@@ -779,13 +779,13 @@ Local roles should only created if a cluster role does not provide the set
 of permissions needed for a particular situation.
 
 Some cluster role names are initially confusing. The cluster role
-`clusteradmin` can be bound to a user using a local role binding, making it appear
+`cluster-admin` can be bound to a user using a local role binding, making it appear
 that this user has the privileges of a cluster administrator. This is not the case.
-The `clusteradmin` cluster role bound to a certain project is more like a super
+The `cluster-admin` cluster role bound to a certain project is more like a super
```

Comment:
Maybe "Binding the \`cluster-admin` role to a certain project grants the permissions of the cluster role..."